extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -336,13 +337,27 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
   /**
    * Executes page level experiment diversion and pushes any experiment IDs
    * onto this.experimentIds.
+   * @param {?string} urlExperimentId
    * @visibleForTesting
    */
-  setPageLevelExperiments() {
+  setPageLevelExperiments(urlExperimentId) {
     if (!isCdnProxy(this.win) && !isExperimentOn(
         this.win, 'expDfpInvOrigDeprecated')) {
       this.experimentIds.push('21060933');
     }
+    if (urlExperimentId) {
+      const experimentId = {
+        // SRA
+        '7': DOUBLECLICK_SRA_EXP_BRANCHES.SRA_CONTROL,
+        '8': DOUBLECLICK_SRA_EXP_BRANCHES.SRA,
+        '9': DOUBLECLICK_SRA_EXP_BRANCHES.SRA_NO_RECOVER,
+
+      }[urlExperimentId];
+      if (experimentId) {
+        this.experimentIds.push(experimentId);
```

Comment:
If you return here then no traffic can be put in the DOUBLECLICK_IDLE_BOOL_EXP experiment.  Can you instead modify the isTrafficEligible for DOUBLECLICK_SRA_EXP to  return false if experiment already selected?

Reply:
Done. When I first made the change, that experiment didn't exist yet, and I didn't notice it when I merged.